html/favicon.html

```diff
@@ -1,25 +1,25 @@
 <!DOCTYPE html>
 <html>
+
 <head></head>
+
 <body>
   <title>My Page Title</title>
   <link rel="icon" type="image/x-icon" href="/images/favicon.ico">
-<h1>Thi is another paragraph</h1>
-
-<p>This is the second paragraph</p>
+  <h1>Thi is another paragraph</h1>
 
-<h2>HTML Image</h2>
-<img src="img_chania.jpg" alt="Flowers in Chania" width="460" height="345">
-<style>
-img {
-  width: 100%;
-}
+  <p>This is the second paragraph</p>
 
-<img src="html5.gif" alt="HTML5 Icon" width="40" height="168">
+  <h2>HTML Image</h2>
+  <img src="img_chania.jpg" alt="Flowers in Chania" width="460" height="345">
```

Comment:
You did not tested this, you don't have img_chaina.jpg in you project. If you don't test you will learn nothing. You cannot commit code that is not working in a real company